mrp_production_auto_post_inventory/models/company.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2016-19 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    mrp_production_auto_post_inventory = fields.Boolean(
+        string="Production Auto Post-Inventory",
+        help="Sets to automatic the post-inventory step in a manufacturing"
+             "order. The inventory will be posted right after the production"
+             "has been done")
```

Comment:
The inventory is always posted when you mark as done the whole MO, so the description might be confusing. I would rather say "the inventory will be posted after some quantity is produced" or something similar.

I would correct this description in all the place it appears (readme, config, here...)